tensorhive/models/RestrictionSchedule.py

```diff
@@ -45,10 +55,21 @@ def check_assertions(self):
         '''
         assert self.hour_end > self.hour_start, 'End hour cannot happen before start hour!'
 
+    @hybrid_property
+    def schedule_days(self):
+        return self._schedule_days
+
     @hybrid_property
     def restrictions(self):
         return self._restrictions
 
+    @schedule_days.setter
+    def schedule_days(self, days: Union[List[Weekday], str]):
+        if isinstance(days, str):
+            self._schedule_days = (''.join(sorted(days)))
+        else:
+            self._schedule_days = self.parse_schedule_list(days)
```

Comment:
Repeated characters in a string or repeated weekdays in a list are not handled. Should we be concerned about it?

Reply:
These are handled in the `is_valid_schedule_expression` method which is invoked when trying to save the schedule to the DB. So while someone might be able to temporarily assign an incorrect schedule to a `_schedule_days` field, he/she would get an assertion exception when attempting to save it.